src/shop.js

```diff
@@ -438,8 +439,11 @@ function BinartaShopjs(checkpoint, deps) {
             if (!fsm.getPaymentProvider())
                 if (shop.localStorage.getItem('binartaJSPaymentProvider'))
                     fsm.setPaymentProvider(shop.localStorage.getItem('binartaJSPaymentProvider'));
-                else
-                    fsm.setPaymentProvider('wire-transfer');
+                else {
+                    var profile = application.profile();
+                    if (profile.availablePaymentMethods && profile.availablePaymentMethods.length == 1)
+                        fsm.setPaymentProvider(profile.availablePaymentMethods[0]);
```

Comment:
I get the impression the old code would automatically select wire-transfer if nothing was remembered.
In your new version it looks to me like it may not select anything at all if there is more than one potential payment method.

That may be ok as nothing will be selected by default then. However I believe one of the reasons we're looking to change this is that some UI's don't communicate the fact no payment provider was selected when attempting to place an order.
So if nothing is selected they may not notice why their order can't be placed.

If you feel like that is how it should work then the UI's should probably communicate properly. therwise we maybe should just select the first one.